### PR TITLE
cleaning ref. metric

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Cleaning the name of reference metric in args and setting pool as nb of CPU

## Before submitting

- [ ] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/metrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to **update the docs**?
- [ ] Did you write any new **necessary tests**?

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/unittests/helpers/testers.py [hunk 1]
```diff
@@ -33,8 +33,13 @@
 except RuntimeError:
     pass
 
-NUM_PROCESSES = 2
-NUM_BATCHES = 4  # Need to be divisible with the number of processes
+if torch.cuda.is_available():
```
Comment (justusschock) [inline tests/unittests/helpers/testers.py:36]: please let's keep it with 2 all the time, everything else will slow things down and to test ddp we only need two processes

**Review by justusschock — CHANGES_REQUESTED** [review]

File: tests/unittests/helpers/testers.py (hunk 1 above)
Comment (Borda) [reply tests/unittests/helpers/testers.py:36]: I have not seen any slowdown... :confused:

**Review by SkafteNicki — APPROVED** [review]
Fine by me, but agree with @justusschock that for simplicity that we should keep the `n_process` and `num_batches` size fixed.

**Comment by Borda** [comment]
> Fine by me, but agree with @justusschock that for simplicity that we should keep the `n_process` and `num_batches` size fixed.

yes, already reverted

**Review by justusschock — APPROVED** [review]

**Pull request merged**